programs/address_map/build.rs

```diff
@@ -0,0 +1,27 @@
+extern crate rustc_version;
+use rustc_version::{version_meta, Channel};
+
+fn main() {
+    // Copied and adapted from
+    // https://github.com/Kimundi/rustc-version-rs/blob/1d692a965f4e48a8cb72e82cda953107c0d22f47/README.md#example
+    // Licensed under Apache-2.0 + MIT
+    match version_meta().unwrap().channel {
```

Comment:
:eyes:

Reply:
Ha, well this is just copy pasta from other build scripts to get the abi digest macro working

Reply:
symlink please? or is some tweak needed?:)

Reply:
Updated to use a symlink instead!